network-api/networkapi/wagtailpages/customblocks.py

```diff
@@ -443,23 +443,26 @@ class LatestProfileList(blocks.StructBlock):
         default=''
     )
 
-    def get_context(self, value, parent_context=None):
+    def get_context(self, value, parent_context=None, no_limit=False, initial_year=False):
```

Comment:
I'm confused by this. 
a.) What is `no_limit`?
b.) Why are we passing initial_year in instead of using the first filter item? (That's what we're going to want to do once we can filter by more than just year anyway)

Reply:
no_limit bypasses the limit query argument so we get "all 2019 entries". We're passing the first filter item, but I've called it year for now because _right now_ it's explicitly set up to deal with years, with https://github.com/mozilla/foundation.mozilla.org/issues/2700 files to expand it to "whatever" instead of just year.